### ISSUE TITLE
Examen 2a

### ISSUE THREAD

**Issue by Jamidd** [PUC-IIC2283/2020-IIC2283-Pruebas#20]
Hola, 
no me queda claro por que esta el b en la probabilidad.
b solo puede tomar valores 0 o 1, y nos piden la probabilidad de obtener b o (0 o 1)??

Gracias,
Jaime.

**Comment by marceloarenassaavedra** [comment]
Dado b en el conjunto {0, 1}, hay que demostrar que:

Pr(LanzarMoneda() retorna b | LanzarMoneda() retorna 0 o 1)  =  1/2

Esto es lo mismo que decir que hay que demostrar dos cosas:

Pr(LanzarMoneda() retorna 0 | LanzarMoneda() retorna 0 o 1)  =  1/2
Pr(LanzarMoneda() retorna 1 | LanzarMoneda() retorna 0 o 1)  =  1/2

Saludos!

**Comment by lsvargas** [comment]
Hola profe, solo para confirmar el simbolo | es un _y_ ?
_Pr(LanzarMoneda() retorna 0 | LanzarMoneda() retorna 0 o 1) = 1/2_  es lo mismo que:
Pr(LanzarMoneda() retorne 0)  = 1/2 y Pr(LanzarMoneda() retorne 0 o 1) = 1/2?

**Comment by marceloarenassaavedra** [comment]
No, el símbolo | es usado para denotar una probabilidad condicional: Pr(A | B) se refiere a la probabilidad de que A suceda dado que sabemos que B sucede.

Saludos!
